warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -8954,7 +8954,7 @@ public static String buildSelector(Metadata metadata, boolean forSearch) {
           sb.append(",");
         }
         encodeName(sb, entry.getKey());
-        if (forSearch && Constants.ABSENT_LABEL_SUPPORT && "".equals(entry.getValue())) {
+        if (forSearch && !Constants.ABSENT_LABEL_SUPPORT && "".equals(entry.getValue())) {
```

Comment:
I understand the change, but in which situation do we enter this code ?  There is never an empty string as label value.